pkg/sshd/server_test.go

```diff
@@ -29,6 +27,27 @@ func TestGitPktLine(t *testing.T) {
 	assert.Equal(t, outStr[4:], str, "remainder of string")
 }
 
+func serverconfigure() (*ssh.ServerConfig, error) {
+	cfg := &ssh.ServerConfig{
+		PasswordCallback: func(c ssh.ConnMetadata, pass []byte) (*ssh.Permissions, error) {
+			// Should use constant-time compare (or better, salt+hash) in
+			// a production setting.
```

Comment:
got it. if you think it's important, can you still file an issue? we can do it in the future